plugins/UsersManager/API.php

```diff
@@ -860,6 +884,18 @@ public function updateUser($userLogin, $password = false, $email = false, $alias
         Piwik::postEvent('UsersManager.updateUser.end', array($userLogin, $passwordHasBeenUpdated, $email, $password, $alias));
     }
 
+    private function verifyPasswordCorrect($userLogin, $password)
+    {
+        /** @var \Piwik\Auth $authAdapter */
+        $authAdapter = StaticContainer::get('Piwik\Auth');
```

Comment:
I'm hoping this works with other auth plugins

Reply:
I haven't looked at your other PR, but will this be an issue for the new two factor auth plugin?

Reply:
I don't think it is. It's only needed for login initially and the password is only a double check here. We will need to update the brute force prevention PR though (lock down IP #2888 ).